JavaScript/packages/recognizers-date-time/src/dateTime/chinese/dateConfiguration.ts

```diff
@@ -227,7 +241,7 @@ export class ChineseDateParser extends BaseDateParser {
             }
             if (!innerResult.success) {
                 // TODO create test
-                innerResult = this.parserDurationWithAgoAndLater(source, referenceDate);
+                innerResult = this.parserDurationWithBeforeAndAfter(source, referenceDate);
```

Comment:
Is this really correct? We don't need both?

Reply:
Or is this more like a rename and the other method is no longer needed?

Reply:
"parserDurationWithAgoAndLater" is used in English and parserDurationWithBeforeAndAfter is for Chinese.

Reply:
Why not unify the method names?